google/bigtable/admin/v2/bigtable_admin_gapic.yaml

```diff
@@ -0,0 +1,300 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.bigtable.admin.spi.v2
+  python:
+    package_name: google.cloud.gapic.bigtable.admin.v2
+  go:
+    package_name: cloud.google.com/go/bigtable/admin/apiv2
+  csharp:
+    package_name: Google.Bigtable.Admin.V2
+  ruby:
+    package_name: Google::Cloud::Bigtable::Admin::V2
+  php:
+    package_name: Google\Cloud\Bigtable\Admin\V2
+  nodejs:
+    package_name: admin.v2
+license_header:
+  copyright_file: copyright-google.txt
+  license_file: license-header-apache-2.0.txt
+interfaces:
```

Comment:
I didn't add resource names because the main bigtable doesn't have them yet - I thought it would make sense to add them together.